charts/datastore/templates/service.yaml

```diff
@@ -17,12 +17,12 @@ spec:
     {{ if or (eq .Values.datastore.serverMode "PLAIN") (eq .Values.datastore.serverMode "AUTOTLS") }} # AUTOTLS mode requires port 80 for communicating with LetsEncrypt
     - protocol: TCP
       port: 80
-      targetPort: 80
+      targetPort: 8080
```

Comment:
How could this possibly be the same port if there are multiple servers on the DOP?

Reply:
WDYM multiple servers on the DOP ?
DOP is one pod with one container

Reply:
It runs multiple servers in AUTOTLS mode. Take a closer look at this part you changed, it would have had both 80 and 443 at the same time - both here in the yaml and in the code. It can't be the same port.